ts/features/walletV3/transaction/components/WalletTransactionDetailsList.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import { View } from "react-native";
+import Placeholder from "rn-placeholder";
+import {
+  HSpacer,
+  IOStyles,
+  ListItemTransaction,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { Transaction } from "../../../../types/pagopa";
+import { formatNumberCentsToAmount } from "../../../../utils/stringBuilder";
+import { Dettaglio } from "../../../../../definitions/pagopa/Dettaglio";
+import {
+  cleanTransactionDescription,
+  getTransactionIUV
+} from "../../../../utils/payment";
+
+type Props =
+  | {
+      transaction?: Transaction | null;
+      loading: boolean;
+      onPress: (operationDetails: Dettaglio) => void;
+    }
+  | {
+      transaction?: Transaction | null;
+      loading: true;
+      onPress: (operationDetails: Dettaglio) => void;
+    };
```

Comment:
why the double props?

Reply:
Whoops, it was an oversight!

Removed into 378a6462f882a1e9d45de757d65a6557cdf786c1